blocksuite/affine/shared/src/commands/model-crud/replace-selected-text-with-blocks.ts

```
@@ -0,0 +1,403 @@
+import { type BlockModel } from '@blocksuite/affine/store';
+import type { Command, TextSelection } from '@blocksuite/std';
+import { match } from 'ts-pattern';
+import { getBlockProps } from '../../utils';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)